### PR TITLE
style(menu table): replace add category button with new bottom row

### PR DESCRIPTION
**Pull request by wangyira**
1.  [Restyle adding a menu category](https://app.asana.com/0/1199117989666292/1199357704571807)
2. Grey out edit button when another window is opened

### CODE REVIEW

File: src/components/dashboard/menu-table/table.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import Checkbox from "./checkbox";
 
 import styled from "styled-components"
 import ArrowIcon from "../../../assets/img/arrow_icon.png"
-
+import PlusIcon from "../../../assets/img/plus-icon.png"
```
Comment (cchyung) [inline src/components/dashboard/menu-table/table.js:7]: Everything looks great except we need a higher resolution plus icon, at the moment it looks to blurry

**Review by cchyung — CHANGES_REQUESTED** [review]

File: src/components/dashboard/menu-table/table.js (hunk 1 above)
Comment (wangyira) [reply src/components/dashboard/menu-table/table.js:7]: I couldn't find the icon asset and took a screenshot of the one in Figma. Let me talk to Ruth and get an icon

**Comment by wangyira** [comment]
@cchyung Just uploaded a new plus icon, thanks!

**Review by cchyung — APPROVED** [review]
looks good!

**Pull request merged**